### PR TITLE
Synthesis Recipes

### PR DESCRIPTION
**Pull request by RwNigma**
Adding Synthesis Recipes:
Candelabrum:
Orichalcum Ring (Desynth)
Peiste Leather
Peiste Leather x3
Angelstone
Painite/Deathstone
Wyrdweave
Angelskin Orb
Oxblood Orb

### CODE REVIEW

File: sql/synth_recipes.sql [hunk 1]
```diff
@@ -60,55 +60,6 @@ CREATE TABLE `synth_recipes` (
 -- Dumping data for table `synth_recipes`
 --
 
-DELIMITER $$
```
Comment (bendangelo) [inline sql/synth_recipes.sql]: Could you add this back please?

Comment (RwNigma) [inline sql/synth_recipes.sql]: Add back in the -DELIMITER $$ as a Comment or as is posted on line 63 there? Sorry I am trying to figure this out from just 1 or two learning sessions of how github works.

Comment (bendangelo) [inline sql/synth_recipes.sql]: Revert it to how it was before without the `-`. You can just copy and paste it back. You'll then need to reimport the table to make sure it doesn't fail the checks.

Ingredients have to be ordered from lowest number to highest or else they won't be recognized.

**Comment by RwNigma** [comment]
Just don't change your files yet. I need to be refreshed on how to add these properly. I had taken from the darkstar recipe and just added mine to the bottom and it seems other things were changed as well?

File: sql/synth_recipes.sql (hunk 1 above)
Comment (RwNigma) [inline sql/synth_recipes.sql]: 6751c8c

Did that update it properly?

**Comment by bendangelo** [comment]
It looks could. Do you know how to squash the commits? It can be done with `rebase --interactive`. 13 commits is way too many, this should only be one commit.

**Comment by RwNigma** [comment]
I honestly have no idea how to do that. I am sorry that it had so many problems. Guess I just figured that a commit when working on it this way was a form of "fixing" if it didn't build properly.

**Comment by bendangelo** [comment]
There must be a tutorial online for squashing commits. It won't take long to do.

**Comment by RwNigma** [comment]
Alright ill look online for it and get it done.

**Comment by xdemolish** [comment]
https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History

**Comment by RwNigma** [comment]
So I would git rebase -i HEAD~13. Then pick 8ab9007 and squash the rest of them? The latest goes at the bottom and the rest get added(squashed) into commit 8ab9007?

**Comment by bendangelo** [comment]
Here is my workflow.

First find the last commit of the commits you have made. That would be `7d70abd9c2cf01675997983724ec83f677e3c7dd`. I usually use `git log --oneline --graph`.

Now start an interactive rebase. `git rebase -i 7d70abd9c2cf01675997983724ec83f677e3c7dd~1`. `~1` will grab the commit before the given commit.

Now you want to mark everything for squash except the first commit. (You can't mark it for squash either way). Pro tip: Mark them all as `fixup` this will squash and not append the commit's message to the final commit message.

Save and then force push to remote. `git push -f origin branch-name`.

**Comment by teschnei** [comment]
^ does the same thing as ~1, I think. An entire keystroke saved

**Comment by RwNigma** [comment]
I will attempt this again tonight. Got busy. If anything I could close this out and just send the the very last commit correct? I know that isn't the preferred way, but just seeing if it is an option. You guys have helped me. I just don't want to mess something up.

**Comment by bendangelo** [comment]
Sure you could do that too.

**Comment by RwNigma** [comment]
Closing this so I can send a new request with the newest commit.

**Pull request closed without merging**